agent/api/task.go

```diff
@@ -65,7 +65,11 @@ type Task struct {
 	Containers []*Container
 	// Volumes are the volumes for the task
 	Volumes []TaskVolume `json:"volumes"`
-
+	// CPULimit is a task-level limit for compute resources. A value of 1 means that
```

Comment:
Should this be `VCPULimit`?

Reply:
I don't know. From agent's perspective its just applying a period and a quota. I'll change it if you think its more clear.

Reply:
I'd like it to match whatever we're going to have in the task definition.  If we don't know that yet, go ahead with either one and then we can fix it once we have an answer.